lib/ipc/common.c

```diff
@@ -91,7 +91,7 @@ _heim_ipc_create_cred(uid_t uid, gid_t gid, pid_t pid, pid_t session, heim_icred
     return 0;
 }
 
-#ifndef HAVE_GCD
+#if !defined(HAVE_GCD) && defined(ENABLE_PTHREAD_SUPPORT)
```

Comment:
This will break the build for !defined(HAVE_GCD) && defined(ENABLE_PTHREAD_SUPPORT) because the heim_issemaphore type will be undefined.  The type is used in the prototype of heim_ipc_semaphore_create() at line 102.

Perhaps #ifdef ENABLE_PTHREAD_SUPPORT the 'cond' field in the heim_isemaphore structure.